src/bionics.cpp

```diff
@@ -2014,6 +2015,20 @@ bool Character::can_install_bionics( const itype &type, player &installer, bool
     }
     int chance_of_success = bionic_manip_cos( adjusted_skill + assist_bonus, autodoc, difficult );
 
+    std::vector<std::string> conflicting_muts;
+    for( const trait_id &mid : bioid->canceled_mutations ) {
+        if( has_trait( mid ) ) {
+            conflicting_muts.push_back( mid->name() );
+        }
+    }
+
+    if( !conflicting_muts.empty() &&
+        !query_yn(
+            _( "This bionic conflicts with the following traits: %s.  Continue anyway?" ),
```

Comment:
Suggest adding something similar to this: "Installing this CBM will remove conflicting traits"